client/modules/onboarding/src/OnboardingStatus/OnboardingStatus.tsx

```diff
@@ -30,6 +30,14 @@ const getContents = (step: TOnboardingStep) => {
   if ('customStatus' in step) {
     return <Tag color={color}>{step.customStatus}</Tag>;
   }
+
+  if (
+    step.events &&
+    step.events[0] &&
+    step.events[0].message === 'Portal access request has not been approved.'
```

Comment:
We can combine these conditionals:
```suggestion
    step.events?.[0]?.message === 'Portal access request has not been approved.'
```